### ISSUE TITLE
Member variables set in child class ignored for taint flow

### ISSUE THREAD

**Issue by LukasReschke** [vimeo/psalm#4605]
This is currently not detected.

https://psalm.dev/r/0b40f43f27

```php
<?php //--taint-analysis	

class TemplateResponse {
    public $taint = '';

    public function getTaint() : string {
        return $this->taint;
    }
}

class StandaloneTemplateResponse extends TemplateResponse {
    public function __construct($taint) {
        $this->taint = $taint; 
    }
}

$extended_response = new StandAloneTemplateResponse($_GET['bar']);
echo $extended_response->getTaint();
```
